contrib/builder/rpm/centos-6/Dockerfile

```diff
@@ -0,0 +1,15 @@
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/generate.sh"!
```

Comment:
The idea behind committing them is that we could then actually consider
making them automated builds and putting them in our set of frozen images.
:+1: